test/fixtures/04-translations/translations/hooks.ts

```diff
@@ -1,3 +1,3 @@
-import('./v1/hooks.js');
```

Comment:
I handled these via https://github.com/balena-io/pinejs/pull/790 instead as that approach makes more sense imo and will also shrink this PR a bit